app/models/miq_group.rb

```diff
@@ -123,56 +125,19 @@ def self.get_httpd_groups_by_user(user)
   end
 
   def get_filters(type = nil)
-    if type
-      (filters.respond_to?(:key?) && filters[type.to_s]) || []
-    else
-      filters || {"managed" => [], "belongsto" => []}
-    end
+    entitlement.try(:get_filters, type)
   end
 
   def has_filters?
-    get_managed_filters.present? || get_belongsto_filters.present?
+    entitlement.try(:has_filters?) || false
   end
 
   def get_managed_filters
-    get_filters("managed")
+    entitlement.try(:get_filters, "managed") || []
```

Comment:
should this be `entitlement.try(:get_managed_filters) || []`

I'd like to limit the places we hardcode "managed" and "belongsto" if possible.